content/en/tracing/llm_observability/trace_an_llm_application.md

```diff
@@ -12,79 +12,172 @@ LLM Observability is not available in the US1-FED site.
-1. Explore the resulting traces on the [LLM Observability traces page][2], and the resulting metrics on the out-of-the-box [LLM Observability dashboard][14].
+{{< tabs >}}
+{{% tab "Decorators" %}}
+{{< code-block lang="python" >}}
+from ddtrace.llmobs.decorators import workflow
 
-Optionally, you can also:
+@workflow
+def process_message():
+    ... # user application logic
+    return
+{{< /code-block >}}
+{{% /tab %}}
 
-- Associate multiple interactions with one user by specifying a `session_id`. See [Tracking user sessions][6] in the SDK documentation.
-- [Persist a span between contexts or scopes][8] by manually starting and stopping it.
-- [Override the name of the LLM application with a different name][9] when starting a root span, which can be useful for differentiating between services or running an experiment.
-- Submit feedback from the users of your LLM application (thumbs up/thumbs down, rating from 1 to 5, and so on) as custom [evaluations][18] via the [SDK][15] or the [API][16]. This allows you to visualize the feedback in your traces and also in the [cluster map view][17], where you can monitor any patterns in different topics of your LLM applications against these evaluations.
+{{% tab "Inline" %}}
+{{< code-block lang="python" >}}
+from ddtrace.llmobs import LLMObs
 
-### Span creation example
+def process_message():
+    with LLMObs.workflow() as span:
+        ... # user application logic
+    return
+{{< /code-block >}}
+{{% /tab %}}
+{{< /tabs >}}
 
-To create a span, use the LLM Observability SDK's `llmobs.decorators.<SPAN_KIND>()` as a function decorator, replacing `<SPAN_KIND>` with the desired [span kind][4].
+### Nesting spans
 
-The example below creates a workflow span:
+Starting a new span before the current span is finished will automatically trace a parent-child relationship between the two spans. The parent span represents the larger operation while the child span represents a smaller nested sub-operation within it.
 
+The examples below create a trace with two spans.
+
+{{< tabs >}}
+{{% tab "Decorators" %}}
 {{< code-block lang="python" >}}
 from ddtrace.llmobs.decorators import workflow
 
+@workflow
+def process_message():
+    perform_preprocessing()
+    ... # user application logic
+    return
+
+@task
+def perform_preprocessing():
+    ... # user application logic
+    return
+{{< /code-block >}}
+{{% /tab %}}
+
+{{% tab "Inline" %}}
+{{< code-block lang="python" >}}
+from ddtrace.llmobs import LLMObs
+
+def process_message():
+    with LLMObs.workflow(name="process_message") as workflow_span:
+        with LLMObs.task(name="perform_preprocessing") as task_span:
+            ... # user application logic
+    return
+{{< /code-block >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+### Annotating spans
+
+To [annotate a span][13], use the LLM Observability SDK's `LLMObs.annotate()` method.
```

Comment:
How about "to add extra information to a span"? right now saying "to annotate a span use the annotate method" is maybe not as informative